tsl/test/expected/remote_txn_resolve.out

```diff
@@ -137,6 +137,146 @@ SELECT count(*) FROM _timescaledb_catalog.remote_txn;
      1
 (1 row)
 
+SELECT create_records();
+ create_records 
+----------------
+ 
+(1 row)
+
+--inject errors in the GID and test "commit" resolution for it
+SET timescaledb.debug_inject_gid_error TO 'commit';
+--heal should error out and the prepared transaction should still be visible
+SELECT _timescaledb_internal.remote_txn_heal_data_node((SELECT OID FROM pg_foreign_server WHERE srvname = 'loopback2'));
```

Comment:
It isn't really clear what we are testing here. It seems as if we are just failing all the healings we are trying, which means we are actually just not executing the healing code. 

Wouldn't it be more interesting to test a failure in healing one transaction out of several transaction, but seeing that the rest succeeds? Or is that what is happening, although it is not clear to me?

Reply:
The workings of the healing code is tested earlier on many times. Here we are trying to exercise the code paths which will fail very rarely via this GUC. Yeah, all heals will fail due to the GUC if there are more than one entries returned from the remote side. But I am not sure that extra effort to test that is worth it.

Reply:
Couldn't one add a counter or boolean in the debug code to fail just the first prepared transaction instead of all of them?

Reply:
I added the above code. Now the first prepared transaction fails and the other one gets healed up as desired.